samples/tcp-echo/tcp-echo.yaml

```diff
@@ -35,6 +35,9 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: tcp-echo
+  labels:
+    app: tcp-echo
+    version: v1
```

Comment:
Name and app

Reply:
Removed based on https://github.com/istio/istio/pull/27242#issuecomment-692549416